src/to-response.ts

```diff
@@ -24,6 +24,38 @@ function isJSONSerializable(value: any) {
 	);
 }
 
+function safeStringify(
+	obj: any,
+	replacer?: (key: string, value: any) => any,
+	space?: string | number,
+): string {
+	const seen = new WeakSet();
```

Comment:
```ts
let id = 0;
const seen = new WeakMap<object, number>(); // ref -> counter

// then

if (seen.has(value)) {
  return `[Circular ref${seen.get(value)}]`;
}
seen.add(value, id++);
```

By doing like this, it would be useful for cross-boundary and recover the circular reference to the original JSON (in the future)